sympy/algebras/tests/test_quaternion.py

```diff
@@ -79,7 +80,7 @@ def test_quaternion_functions():
     assert integrate(Quaternion(x, x, x, x), x) == \
     Quaternion(x**2 / 2, x**2 / 2, x**2 / 2, x**2 / 2)
 
-    assert Quaternion.rotate_point((1, 1, 1), q1) == (1 / 5, 1, 7 / 5)
+    assert Quaternion.rotate_point((1, 1, 1), q1) == (S(1)/5, 1, S(7)/5)
```

Comment:
Earlier it didn't had the `from __future__ import divison`. How was it passing the python2 tests earlier?

`(1 / 5, 1, 7 / 5)` would have reduced to `(0, 1, 1)`, wouldn't it?

Reply:
@gxyd Yes it wasn't passing the tests earlier.

Reply:
But how did it pass Travis CI tests?

Reply:
It even passes tests locally with python2. Seems like even without an explicit import of `division` from `__future__` it is getting `divison` from somewhere else.

Reply:
@RohitJain1103 you say, that it wasn't passing the tests earlier. On which version of python and where (locally or travis) was it not passing the tests?

Reply:
@gxyd On Python 2.7.11 locally it wasn't passing the tests. But this was when I started this PR.

Reply:
So that means on your master branch, with python2 you are still getting this error? If you can reproduce the error, please comment here the error you were getting.

Reply:
On my master branch I am getting all tests passing, but all these changes seem to be included in test_quaternion after pulling from sympy/master.

Reply:
If I understand you correctly. These changes are not there in sympy master. See https://github.com/sympy/sympy/blob/master/sympy/algebras/tests/test_quaternion.py . At the top, it doesn't has `from __future__ import division`.

Reply:
Sorry I hadn't done the pulling well.
After pulling I am getting the same tests failing which are as follows:-
![capture](https://user-images.githubusercontent.com/20405436/34152617-3d6c2eac-e4d5-11e7-8394-ba74e2545635.PNG)
![capture1](https://user-images.githubusercontent.com/20405436/34152637-47fc05f4-e4d5-11e7-8f18-8f1e246a2907.PNG)

Reply:
`division` is imported in https://github.com/sympy/sympy/blob/3707b6922a7f9d503873f1800bbcf4de6a61e220/sympy/utilities/runtests.py#L15
and per [PEP 238](https://www.python.org/dev/peps/pep-0238/), it is applied to tests.

@asmeurer Is this the expected behavior?

Reply:
ping @asmeurer.